src/custom_modules/DAO_modules/donors.ts

```diff
@@ -230,17 +231,18 @@ async function search(filter): Promise<Array<Donor>> {
 //region Add
 /**
  * Adds a new Donor to the database
- * @param {Donor} donor A donorObject with two properties, email (string) and name(string)
  * @returns {Number} The ID of the new Donor if successfull
  */
-async function add(email = "", name, newsletter = null) {
+async function add(
```

Comment:
This could be used quite a few places, are we sure we are sending the right params everywhere? :)

Reply:
Yes, I changed each usage of `DAO.donors.add` to use an object as argument